bigtable/internal/testproxy/proxy_test.go

```diff
@@ -0,0 +1,201 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+	"os"
+
+	"testing"
+
+	"cloud.google.com/go/bigtable"
+	"cloud.google.com/go/bigtable/bttest"
+	pb "github.com/googleapis/cloud-bigtable-clients-test/testproxypb"
+	"google.golang.org/api/option"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/test/bufconn"
+)
+
+const (
+	buffer           = 1024 * 1024
+	tableName        = "table"
+	columnFamily     = "cf"
+	testProxyClient  = "testProxyClient"
+	testProxyAddress = "localhost:9990"
+	bigtableAddress  = "localhost:9999"
+	rowKey           = "row"
+)
+
+var (
+	lis    *bufconn.Listener
+	client *pb.CloudBigtableV2TestProxyClient
+)
+
+func bufDialer(context.Context, string) (net.Conn, error) {
+	return lis.Dial()
+}
+
+// helper function to populate the in-memory BT table.
+// TODO(developer): Expose this (and bttest) as suite of test tools
```

Comment:
Does this mean the test proxy maintainers should do this? Or someone else? TODO(developer) usually means non-google user of the package. Perhaps just `TODO:`?

Reply:
I had intended to leave this comment for future maintainers ... but this is an inappropriate place for such a reminder.

Removed.